requirements-dev.txt

```diff
@@ -0,0 +1,8 @@
+black==22.8.0
+flake8==4.0.1
+flake8-bugbear==22.7.1
+mypy==0.971
+sphinx==4.3.2
+sphinx_mdinclude==0.5.1
+ufmt==2.0.1
+usort==1.0.5
```

Comment:
Why have these? Shouldn't they live in `pyproject.toml`?

Reply:
lol, oops, my commit testing thx on the project snuck in 😅